src/test/java/com/mongodb/jdbc/MongoSQLMock.java

```diff
@@ -284,4 +358,191 @@ MongoJsonSchemaResult generateSchema() {
         schemaResult.schema.mongoJsonSchema = generateMongoJsonSchema();
         return schemaResult;
     }
+
+    protected static MongoJsonSchema generateMongoJsonSchemaAllTypes() {
+        String schema =
+                ""
```

Comment:
Nit: I think this empty string can be safely removed. Or does it come from Spotless?